test/parallel/test-child-process-execfile.js

```diff
@@ -99,3 +100,23 @@ const execOpts = { encoding: 'utf8', shell: true };
   });
   execFile(process.execPath, [fixture, 0], { signal }, callback);
 }
+
+// Verify the execFile() stdout is the same as execFileSync().
+{
+  const file = 'echo';
+  const args = ['foo', 'bar'];
+
+  // Test with and without `{ shell: true }`
+  [
+    // Skipping shell-less test on Windows because …
```

Comment:
Because…? I was expecting you'd finish that comment with a justification 😅

Reply:
Github's adoption suggestions do not allow me to add content directly.
I have done in a later commit